builder/src/build/patches/investigated/update-webpack-chunks-file/get-chunk-installation-identifiers.ts

```diff
@@ -0,0 +1,77 @@
+import * as ts from "ts-morph";
+
+export async function getChunkInstallationIdentifiers(sourceFile: ts.SourceFile): Promise<{
+  installedChunks: string;
+  installChunk: string;
+}> {
+  const installChunkDeclaration = getInstallChunkDeclaration(sourceFile);
+  const installedChunksDeclaration = getInstalledChunksDeclaration(sourceFile, installChunkDeclaration);
+
+  return {
+    installChunk: installChunkDeclaration.getName(),
+    installedChunks: installedChunksDeclaration.getName(),
+  };
+}
+
+function getInstallChunkDeclaration(sourceFile: ts.SourceFile) {
```

Comment:
nits: 
- add return type
- Add a snippet of the code we're looking for in a function level comment